core/lib/spree/testing_support/factories/data_feed_setting_factory.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :data_feed_setting, class: Spree::DataFeedSetting do
```

Comment:
in migration I saw `store` reference, maybe it's good idea to add here also 
```
store { create(:store) }
```

wdyt ?

Reply:
Seems like a good idea.